### ISSUE TITLE
in [eslint] Cannot use 'in' operator to search for 'type' in undefined

### ISSUE THREAD

**Issue by vaynevayne** [sweepline/eslint-plugin-unused-imports#104]
```jsx
import type { CanvasEvents, TPointerEvent, TPointerEventInfo } from 'fabric6'
```

**Comment by sweepline** [comment]
You reported that its the same issue as #98. Since you did not give any extra information to help debug this I will close it. If it is not code from this repository emitting the error, you should report it upstream.
